src/annotator/sidebar.tsx

```diff
@@ -79,15 +79,20 @@ function createSidebarIframe(config: SidebarConfig): HTMLIFrameElement {
 
   const sidebarFrame = document.createElement('iframe');
 
-  // Enable media in annotations to be shown fullscreen
-  sidebarFrame.setAttribute('allowfullscreen', '');
-
   sidebarFrame.src = sidebarAppSrc;
   sidebarFrame.title = 'Hypothesis annotation viewer';
   sidebarFrame.className = 'sidebar-frame';
-  sidebarFrame.allow = 'clipboard-write';
 
-  return sidebarFrame;
+  // Enable media in annotations to be shown fullscreen, and allow copying to
+  // the clipboard.
+  sidebarFrame.allow = 'fullscreen; clipboard-write';
```

Comment:
From what I can tell from following the spec discussions, the `clipboard-write` property is deprecated, but Chrome still requires it. _If_ Chrome removes their checks, we could remove this in 2-3 years time.